cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -288,6 +290,127 @@ var _ = Describe("validation", func() {
+					}
+				}
+				return false
+			}()
+			Expect(n3000NicExists).To(BeTrue(), "No Intel fpga n3000 NICs are presented on the n3000nodes")
+		})
+
+		It("should have a ready deployment for the OpenNESS Operator for Intel FPGA PAC N3000 (Programming) operator", func() {
+			deployment, err := testclient.Client.Deployments(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DeploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(deployment.Status.ReadyReplicas).To(Equal(deployment.Status.Replicas), "Deployment n3000-controller-manager is not ready")
+		})
+
+		It("should have all the required OpenNESS Operator for Intel FPGA PAC N3000 (Programming) operands", func() {
+			n3000Nodes, err := testclient.Client.Nodes().List(context.Background(), metav1.ListOptions{LabelSelector: utils.N3000NodeLabel})
+			Expect(err).ToNot(HaveOccurred())
+			if len(n3000Nodes.Items) < 1 {
+				Skip("Cluster doesn't have n3000 node")
+			}
+			numberN3000Nodes := int32(len(n3000NodeList.Items))
+			daemonsetDriver, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetDriverName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			daemonsetTelemetry, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetTelemetryName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			daemonsetN3000Daemon, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetN3000DaemonName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			daemonsetN3000Discovery, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetDiscoveryName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(daemonsetDriver.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetDriverName))
+			Expect(daemonsetTelemetry.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetTelemetryName))
+			Expect(daemonsetN3000Daemon.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetN3000DaemonName))
+			Expect(daemonsetN3000Discovery.Status.NumberReady).To(BeNumerically(">", 0), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetDiscoveryName))
+		})
+
+		It("should have the n3000 CRDs available in the cluster", func() {
+			crd := &apiext.CustomResourceDefinition{}
+			err := testclient.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.N3000NodeCRDName}, crd)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = testclient.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.N3000ClusterCRDName}, crd)
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
+
+	Context("OpenNESS Operator for Intel® FPGA PAC N3000 (Management)", func() {
```

Comment:
sorry can you please change this to `sriov-fec` we need this to skip the context